Kernel/Bus/USB/USBDMAPool.h

```diff
@@ -50,8 +50,9 @@ class USBDMAPool {
         SpinlockLocker locker(m_pool_lock);
 
         // We're out of buffers!
-        if (m_free_buffer_stack.is_empty())
+        if (m_free_buffer_stack.is_empty()) {
             return nullptr;
```

Comment:
This change doesn't match the coding style, no { } around single line if statements plz